packages/charting/src/components/BenchmarkBarChart/BenchmarkBarChart.style.ts

```diff
@@ -0,0 +1,37 @@
+import { IBenchmarkBarChartStyleProps, IBenchmarkBarChartStyle } from './BenchmarkBarChart.types';
+
+export const getBenchmarkBarChartStyles = (props: IBenchmarkBarChartStyleProps): IBenchmarkBarChartStyle => {
+  const { width, benchmarkColor } = props;
+
+  return {
+    root: {
+      width: width || 300,
+      fontFamily: 'Segoe UI',
```

Comment:
>fontFamily: 'Segoe UI', [](start = 6, length = 23)

use fabric fonts with fallback

Reply:
Set fontFamily value to FontFamilyFallbacks `'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif`

fontFamily: `'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif`

Reply:
Fabric has defaultFont that will give you this fontfalmily, you don't have to copy that here. If fabric changes things you will get it for free

---
In reply to: [283624394](https://github.com/OfficeDev/office-ui-fabric-react/pull/9039#discussion_r283624394) [](ancestors = 283624394)

Reply:
I have no idea how to use the defaultFont. I just found **FontFamilyFallbacks** in the font.ts file. This variable is not exported and I can't use it in BenchmarkBarChart.style.ts file. Could you give me some suggestion on how to use it?

Reply:
you can get at a fonts object that is passed in through the theme prop.

Reply:
it would look like this
`props.theme.fonts.medium`

Reply:
Change it to     
root: [
      fonts.medium,
      {
        width: width || 300,
        fontSize: '13px'
      }
    ],